res/values-vi/strings.xml

```diff
@@ -1308,6 +1397,19 @@ Lệnh:
     <string name="no_response">Không có phản hồi từ máy chủ. Đề nghị kiểm tra kết nối mạng.</string>
     <string name="RAMADAN">RAMADAN</string>
     <string name="SUDDEN_ATTACK">TẤN CÔNG BẤT NGỜ</string>
+    <string name="sign_in_explanation" translatable="true">
+"Sign in to access multiplayer. With an account, you may be able to:
+
+- Play and communicate with the players around the world;
+- Level-up, unlock achievements and complete daily tasks for rewards;
+- Higher variety of skins since default, level, season, achievement, pets, hats, halos and packs;
+- Share your progress with third-party;
+- Create, manage and decorate your profile;
+- Purchase plasma at the shop;
+- Upload skins to the community and purchase user-generated skins.
+
+If you do not wish to sign in and enjoy the advantages, you might choose playing on single-player instead."
+    </string>
```

Comment:
This one is not completely based on `res/values`: Included 2 features that are unlockable by having an account for more clarity, and improved reading for untranslated.